### PR TITLE
Google Drive Config for WIP

### PR DESCRIPTION
**Pull request by joinimran**
Added a use case where user can block Google Drive not to sync WIP protected files.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4342

### CODE REVIEW

**Comment by jvsam** [comment]
@nenonix & @JohanFreelancer9 for copy review please. Thank you.

File: windows/security/information-protection/windows-information-protection/testing-scenarios-for-wip.md [hunk 1]
```diff
@@ -172,6 +172,17 @@ You can try any of the processes included in these scenarios, but you should foc
             </ul>
         </td>
     </tr>
+    <tr>
+        <td>Stop Google Drive to sync WIP protected files and folders.</td>
```
Comment (illfated) [inline windows/security/information-protection/windows-information-protection/testing-scenarios-for-wip.md]:
```suggestion
        <td>Stop Google Drive from syncing WIP protected files and folders.</td>
```

File: windows/security/information-protection/windows-information-protection/testing-scenarios-for-wip.md [hunk 2]
```diff
@@ -172,6 +172,17 @@ You can try any of the processes included in these scenarios, but you should foc
...
+        <td>
+            <ul>
+                <li>In silent configuration add Google Drive in Protected Apps and set it to Deny. This way Google Drive will not sync                       WIP protected files and folders.</li>
```
Comment (illfated) [inline windows/security/information-protection/windows-information-protection/testing-scenarios-for-wip.md]:
```suggestion
                <li>In silent configuration add Google Drive in Protected Apps and set it to Deny. This way Google Drive will not sync WIP protected files and folders.</li>
```

**Comment by illfated** [comment]
Both of my suggested changes could possibly be improved even more, even though I guess this update is more about publicizing the new information.

**Comment by joinimran** [comment]
Suggestions applied.

File: windows/security/information-protection/windows-information-protection/testing-scenarios-for-wip.md [hunk 3]
```diff
@@ -172,6 +172,17 @@ You can try any of the processes included in these scenarios, but you should foc
             </ul>
         </td>
     </tr>
+    <tr>
+        <td>Stop Google Drive from syncing WIP protected files and folders.</td>
+        <td>
+            <ul>
+                <li>In silent configuration, add Google Drive in Protected Apps and set it to Deny. This way, Google Drive will not sync WIP protected files and folders.</li>
```
Comment (nenonix) [inline windows/security/information-protection/windows-information-protection/testing-scenarios-for-wip.md]:
```suggestion
                <li>In silent configuration, add Google Drive to Protected Apps and set it to Deny. This way, Google Drive will not sync WIP protected files and folders.</li>
```

Comment (jvsam) [reply windows/security/information-protection/windows-information-protection/testing-scenarios-for-wip.md]: @joinimran please check. Thanks.

**Comment by joinimran** [comment]
Thanks @nenonix for fixing this which makes more sense.

@jvsam Over to you now.

**Comment by jvsam** [comment]
Hello @DulceMontemayor and @Dansimp, this pull request has been copy edited and is ready for your final review and approval. Please let us know if there are outstanding issues to be resolved prior to merge. Thank you. See: [file changes](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4493/files).

**Comment by jvsam** [comment]
Thank you for merging this PR @Dansimp! Happy weekend!

**Pull request merged**